Algorithms-Merge-Sort.md

```diff
@@ -13,36 +13,124 @@ To learn about Merge Sort, a basic knowledge about [Recursion](http://programmer
 The biggest advantage of using Merge sort is that the [time complexity](https://www.youtube.com/watch?v=V42FBiohc6c&list=PL2_aWCzGMAwI9HK8YPVBjElbLbI3ufctn) is only n*log(n) to sort an entire Array. It is a lot better than n^2 running time of bubble sort or insertion sort.
 Before we write the JavaScript code, let us understand how merge sort works with the help of a diagram.
 
-![alt tag] (https://i67.tinypic.com/2ahe49y.png)
+![Merge Sort](https://i67.tinypic.com/2ahe49y.png)
```

Comment:
Please change the link to https://i.imgur.com/ac98ccu.png

That link is not working, I uploaded the image to the one I just posted.